prisma/migrations/20260704063117_add_default_card_self_relation/migration.sql

```
@@ -0,0 +1,58 @@
+-- AlterTable
+ALTER TABLE "card" ADD COLUMN     "defaultCardId" TEXT;
+
+-- CreateIndex
+CREATE INDEX "card_defaultCardId_idx" ON "card"("defaultCardId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)